apisix/cli/ngx_tpl.lua

```diff
@@ -69,10 +69,10 @@ lua {
 {% if enabled_stream_plugins["prometheus"] and not enable_http then %}
 http {
     lua_package_path  "{*extra_lua_path*}$prefix/deps/share/lua/5.1/?.lua;$prefix/deps/share/lua/5.1/?/init.lua;]=]
-                       .. [=[{*apisix_lua_home*}/?.lua;{*apisix_lua_home*}/?/init.lua;;{*lua_path*};";
+    .. [=[{*apisix_lua_home*}/?.lua;{*apisix_lua_home*}/?/init.lua;;{*lua_path*};";
```

Comment:
Could you disable the auto-format in your editor? This feature breaks a lot of indentations.